detection-rules/attachment_html_smuggling_embedded_b64_pe.yml

```diff
@@ -0,0 +1,28 @@
+name: "Attachment: HTML smuggling with embedded base64-encoded executable"
+description: |
+  HTML attachmemt contains a base-64 encoded executable.
+type: "rule"
+severity: "high"
+  type.inbound 
+  and any(attachments,
+      (
+          .file_extension in~ ("html", "htm") or
+          .file_extension in~ $file_extensions_common_archives or
+          .file_type == "html"
+      ) 
+      and any(beta.binexplode(.), 
+        any(.flavors.yara, . == 'base64_pe')) 
+  )
+  and (
+          (
+              sender.email.domain.root_domain in $free_email_providers
+              and sender.email.email not in $recipient_emails
+          )
+          or (
+              sender.email.domain.root_domain not in $free_email_providers
+              and sender.email.domain.domain not in $recipient_domains
+          )
+  )
```

Comment:
sorry my last comment was confusing! i don't think we need this snippet for this particular rule based on how great it performed in our Hunts. but if we did, this snippet in general would be better than the original `and sender.email.email not in $recipient_emails`

Reply:
ah ok, no worries! Do we want to go back to what was there, or just keep this one in?

Reply:
i think we can delete this block

Reply:
this is what i was thinking [cc8929d](https://github.com/sublime-security/sublime-rules/pull/182/commits/cc8929dd9309ad463e0e415c13f4324b41ea77f6)